### ISSUE TITLE
Can char cnn support masking?

### ISSUE THREAD

**Issue by Das-Boot** [CyberZHG/keras-word-char-embd#3]
Nice work! But when I use char cnn, I must set `char_mask_zero=False`, otherwise the code will report an error: Layer conv1d does not support masking... Can we use char cnn that supports masking?

**Comment by Crista23** [comment]
Hi @Das-Boot ! With what version of Keras and Tensorflow did you manage to get the code running?

**Comment by Das-Boot** [comment]
Keras==2.2.4 Tensorflow==1.12.0

**Comment by CyberZHG** [comment]
https://github.com/CyberZHG/keras-word-char-embd/blob/cca6ddff01b6264dd0d12613bb9ed308e1367b8c/tests/test_get_embedding_layer.py#L77-L86

**Comment by Das-Boot** [comment]
Thanks！
